### PR TITLE
Adding prober reject settings and examples.

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: pkg/eventshub/prober.go [hunk 1]
```diff
@@ -136,6 +153,31 @@ func (p *EventProber) SenderDone(prefix string) feature.StepFn {
...
+// ReceiverDone will poll until the receiver has received all expected events.
+func (p *EventProber) ReceiverDone(from, to string) feature.StepFn {
+	return func(ctx context.Context, t feature.T) {
+		interval, timeout := environment.PollTimingsFromContext(ctx)
+		err := wait.PollImmediate(interval, timeout, func() (bool, error) {
+			sent := p.SentBy(ctx, from)
+			fmt.Println(from, "has sent", len(sent))
+
+			received := p.ReceivedBy(ctx, to)
+			fmt.Println(to, "has received", len(received))
```
Comment (vaikas) [inline pkg/eventshub/prober.go:165]: Sure would be nice to plumb the logger here so we'd at least get some timing information. 🤔

File: pkg/eventshub/prober.go [hunk 2]
```diff
@@ -221,25 +285,40 @@ func (p *EventProber) SenderEventsFromSVC(svcName, path string) feature.StepFn {
...
+// Warning: only call once.
 func (p *EventProber) SenderFullEvents(count int) {
-	for i := 0; i < count; i++ {
+	event := cetest.FullEvent()
+	if count == 1 {
 		id := uuid.New().String()
-		event := cetest.FullEvent()
 		event.SetID(id)
-		p.ids = append(p.ids, id)
+		p.ids = []string{id}
```
Comment (vaikas) [inline pkg/eventshub/prober.go:294]: This is kind of wonky that you have to special case sending 1 event here and in SenderMinEvents.

Comment (n3wscott) [reply pkg/eventshub/prober.go:294]: it is a limitation of eventshub :idk:, I assumed you could chain these together originally, and that is not true.

File: pkg/eventshub/prober.go (hunk 1 above)
Comment (n3wscott) [reply pkg/eventshub/prober.go:165]: that was debug, I can remove now or in a followup. We should have a better story about the logger.

File: pkg/eventshub/prober.go (hunk 2 above)
Comment (vaikas) [reply pkg/eventshub/prober.go:294]: Can we create an issue for it?

**Review by vaikas — APPROVED** [review]
/lgtm

**Pull request merged**